keyboards/keebio/quefrency/keymaps/milouk/keymap.c

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2023 Michael Loukeris
```

Comment:
This file should be removed if you intend to use the `keymap.json`. (That's what takes precedence anyway)